### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by brendaloya**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,40 +22,64 @@ Just like on Ethereum, all Tellor contracts on Alogrand are open-source, freely
...
+- Fund a **deployment** account with 16 ALGO to test (this account will be used for deploying a medianizer and 5 data feed contracts). Algorand addresses have to be funded with a minimum of 15 ALGO but additional is needed to deploy and transact.
+
+**1. Run docker and leave it running**
+
+**2. Create .env file**
+
+This sets up accounts to be used securely since the .env file is not pushed to github. You can use the `.env.example` as an example.
+
+- Create a file called .env
+- Copy and paste the variable names from the `.env.examplea` file and add your mnemonics to the .env file
```
Comment (tallywiesenberg) [inline README.md]: typo

**Review by tallywiesenberg — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (brendaloya) [reply README.md]: thanks. can you just merge and fix it? 
Also, can you expand on how this option (-tf 120) ensures the data is fresh.

Comment (tallywiesenberg) [reply README.md]: updated!

Comment (tallywiesenberg) [reply README.md]: oh and i fixed the typo

**Pull request merged**